packages/@ember/string/index.ts

```diff
@@ -111,6 +112,20 @@ function _fmt(str: string, formats: any[]) {
   @public
 */
 export function loc(str: string, formats: any[]): string {
+  deprecate(
+    'loc is deprecated, please use a dedicated localization solution like ember-intl. More alternatives listed at https://emberobserver.com/categories/internationalization.',
+    false,
+    {
+      id: 'ember-string.loc',
+      until: '4.0.0',
+      for: '@ember/string',
```

Comment:
Should this be `ember-source`? Do I need to update the library version of `@ember/string`? cc @rwjblue

Reply:
Ya, should be `ember-source`. It represents the package users consume that issues the deprecation (which is ember-source).

Reply:
Updated.